### PR TITLE
Add code of conduct and reporting guidelines

### PR DESCRIPTION
**Pull request by annajung**
### Description

- Add a copy of code of conduct from https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md with changes to the email address used.
- Add a copy of reporting guidelines from https://github.com/anitab-org/mentorship-backend/blob/develop/docs/reporting_guidelines.md with changes to the email address used.

note: code of conduct references reporting guidelines

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: reporting_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. 
+* If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link.
+* Any extra context you believe existed for the incident.
+* If you believe this incident is ongoing.
+* Any other information you believe we should have.
+
+## What happens after you file a report?
+
+You will receive an email from the Anitab.org Open Source's Code of Conduct response team acknowledging receipt as soon as possible, but within 48 hours.
```
Comment (renana) [inline reporting_guidelines.md:23]: Open Source Day's

File: reporting_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Once we've determined our final action, we'll contact the original reporter to let them know what action (if any) we'll be taking. We'll take into account feedback from the reporter on the appropriateness of our response, but our response will be determined by what will be best for community safety.
+
+Finally, the response team will make a report on the situation to the Anitab.org's Open Source Board. The board may choose to issue a public report of the incident or take additional actions.
+
+## Appealing the response
+
+Only permanent resolutions (such as bans) may be appealed. To appeal a decision of the working group, contact the community admins at osdghc@anitab.org with your appeal and we will review the case.
+
+_Revision 1.0, DRAFT, proposed June 2014_
```
Comment (renana) [inline reporting_guidelines.md:50]: Should we replace this with today and point to the mentoring source or should we leave this as is?

File: reporting_guidelines.md (hunk 1 above)
Comment (annajung) [reply reporting_guidelines.md]: done + other places with `AnitaB.org's Open Source` to `AnitaB.org's Open Source Day`

File: reporting_guidelines.md (hunk 2 above)
Comment (annajung) [reply reporting_guidelines.md]: Modified to May 2021 with reference to the [AnitaB mentorship reporting guidelines](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/reporting_guidelines.md). Let me know what you think!

**Review by renana — APPROVED** [review]
Approving for now since AnitaB is super busy with GHC-EMEA and it will take time until they will be able to take a look into this.

**Pull request merged**